docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: "3.8"
+services:
+  gofer:
+    image: ghcr.io/chronicleprotocol/gofer-cli:latest
```

Comment:
Just to be sure, if I wanted to run this service directly instead of with docker I would run `gofer agent` right ?

Reply:
Also we probably want to adjust the `README` to include instructions on this.